appveyor.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 0.15.0-SNAPSHOT.{build}-{branch}
+
+shallow_clone: true
+
+platform: x64
+configuration: Debug
+
+cache:
+  - packages
+
+# install section: install maven, protoc etc.
+install:
+  - ps: .\dev\appveyor-install-dependencies.ps1
+
+build_script:
+  - cmd: msbuild .\lang\cs\Org.Apache.REEF.sln /p:Configuration="Debug" /p:Platform="x64" /m
+
+test_script:
+  - cmd: msbuild .\lang\cs\TestRunner.proj /p:Configuration="Debug" /p:Platform="x64"
+#  To run specific .dll only
```

Comment:
No, we have Wiki for that kind of explanation. 
Please leave that [Continuous Integration Setup](https://cwiki.apache.org/confluence/display/REEF/Continuous+Integration+Setup).